platforms/fpga/runtime/src/board.rs

```diff
@@ -592,6 +596,13 @@ pub unsafe fn main() {
     if let Some(exit) = exit {
         crate::io::exit_emulator(exit);
     }
+
+    // Disable WDT1 before running the loop
+    let soc: StaticRef<soc::regs::Soc> =
```

Comment:
I think we don't need to modify Caliptra's own watchdog timers here -- it should take care of disabling them itself.